tableaudocumentapi/datasource.py

```diff
@@ -59,6 +59,29 @@ def from_file(cls, filename):
         dsxml = ET.parse(filename).getroot()
         return cls(dsxml, filename)
 
+    def get_connections_by_server_name(self, server_name):
```

Comment:
Looking for feedback on method name and implementation.

Simple iteration and matching for now, hey, it works.